internal/execute/tableutil/stream.go

```diff
@@ -0,0 +1,184 @@
+package tableutil
+
+import (
+	"context"
+	"sync/atomic"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/internal/errors"
+)
+
+// SendFunc is a function that will be used to construct a stream.
+type SendFunc func(flux.ColReader)
```

Comment:
Can you make the comment more informative? This function is to send a `ColReader`/ TableBuffer to a table stream so it could be read by the table consumer.

I asked because I was confused and did not get what does this function do until I read all the code below in this file.